andela/healthchecks-apollo
# #151948280 User should be able to configure periodic reports

#### What does this PR do?
* Adds a feature User should be able to configure periodic reports, for example monthly, weekly, daily.
#### Description of the task to be completed?
* The dashboard only allows subscribing to monthly report. This should be changed to be user configurable to either be daily, weekly or monthly.
#### How should this manually be tested?
* First run automatic tests from the project root directory and ensure they all pass; 
>python manage.py test hc.accounts.tests_profile 
* To manually test this PR; 
1. Log into the application
2. Navigate to the account settings button at the top right corner of your screen and click on it
3. Navigate to the periodic reports section and click on the configure button. This brings up a modal form where you can select the different reporting periods user can choose from
4. After selecting a reporting period click on the save button, to apply changes. See screenshots section for more details
#### Pivotal Tracker story?
* See [Feature #151948280](https://www.pivotaltracker.com/story/show/151948280)
#### Screenshots (if appropriate)
##### Step 1:
<img width="1260" alt="screen shot 2017-11-02 at 12 14 40" src="https://user-images.githubusercontent.com/6205925/32321860-a46cbb02-bfd3-11e7-830c-5115a8859637.png">

##### Step 2:
<img width="1205" alt="screen shot 2017-11-02 at 12 13 27" src="https://user-images.githubusercontent.com/6205925/32321893-b7b8cc3c-bfd3-11e7-9232-f9faaaec00af.png">

##### Step 3:
<img width="1214" alt="screen shot 2017-11-02 at 12 13 54" src="https://user-images.githubusercontent.com/6205925/32321912-c24fec8e-bfd3-11e7-9a41-23d921d39c67.png">

#### Questions:
